animareflection/ui
# Add Addon and InputElement Options for `Input` and `NumberInput`

## Description

##### Task link: https://trello.com/c/rcygM0ca/203-numberinput-and-input-addons-and-elements

Added optional `leftAddon`, `rightAddon`, `leftInputElement`, and `rightInputElement` props for both `NumberInput` and `Input` where they were missing.

## Test Steps

1) Verify that logic for component is sound.
2) Verify that components render appropriately in storybook and styles look good.
